src/Popover/Popover.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withKnobs, text, select } from '@storybook/addon-knobs'
+import { defaultTemplate } from '../../storybook/decorators/storyTemplates'
+import { Button, OverlayTrigger, Popover } from '../index.js'
+
+const stories = storiesOf('Popover', module)
+const description = (
+  <p>
+    This component is based on React Bootstrap Popover component. See{' '}
+    <a href="https://react-bootstrap.github.io/components.html#popovers">
+      React Bootstrap Docs
+    </a>{' '}
+    for complete Popover component documentation.
+  </p>
+)
+stories.addDecorator(withKnobs)
+stories.addDecorator(
+  defaultTemplate({
+    title: 'Popover',
+    documentationLink:
+      'http://www.patternfly.org/pattern-library/widgets/#popover',
+    description: description
+  })
+)
+
+stories.addWithInfo('Popover', () => {
+  const popover = (
+    <Popover id="popover" title="Popover">
+      <strong>Holy guacamole!</strong> Check this info.
+    </Popover>
+  )
+  const placement = select(
+    'Placement',
+    ['top', 'bottom', 'left', 'right'],
+    'right'
+  )
+  const trigger = text('Trigger', 'hover focus')
```

Comment:
I would use a select knob here w/ options `click`, `hover`, and `focus` instead of a text knob.